multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/madara/MadaraGenerator.kt

```diff
@@ -50,7 +50,7 @@ class MadaraGenerator : ThemeSourceGenerator {
         SingleLang("Atikrost", "https://atikrost.com", "tr", overrideVersionCode = 1),
         SingleLang("Atlantis scan", "https://atlantisscan.com", "es"),
         SingleLang("AZManhwa", "https://azmanhwa.net", "en"),
-        SingleLang("Azora", "https://azoraworlds.net", "ar", overrideVersionCode = 4),
+        SingleLang("Azora", "https://azoranov.com/", "ar", overrideVersionCode = 5),
```

Comment:
Can remove the trailing slash.

```suggestion
        SingleLang("Azora", "https://azoranov.com", "ar", overrideVersionCode = 5),
```